server/config/config.go

```diff
@@ -37,6 +37,7 @@ type RedisConfig struct {
 	Address  string
 	Password string
 	Database int
+	UseTLS   bool `yaml:"use_tls"`
```

Comment:
I added the `yaml` tag here which fixes the config file parsing (and related test).